src/main/java/io/micronaut/gradle/graalvm/MicronautGraalPlugin.java

```diff
@@ -80,7 +83,13 @@ public void apply(Project project) {
                         test.systemProperty("micronaut.test.server.executable", file.getAbsolutePath());
                     });
                     boolean enabled = test.isEnabled() && GraalUtil.isGraalJVM();
-                    nativeImageTestTask.setEnabled(enabled);
+                    nativeImageTestTask.onlyIf(task -> {
+                        boolean isGraal = GraalUtil.isGraalJVM();
+                        if (!isGraal) {
+                            project.getLogger().log(LogLevel.INFO, "Skipping testNativeImage because the configured JDK is not a GraalVM JDK");
+                        }
+                        return isGraal;
```

Comment:
@graemerocher didn't you miss `test.isEnabled()` here?

Reply:
True, will fix